### PR TITLE
[docs-only] Add .adoc table version to env-var-deltas

### PR DESCRIPTION
**Pull request by mmattel**
@dragonchaser has created an envvar diff table based on markdown.

To consume this table in our documentation, we need to create a .adoc table from it.

This PR provides:
* A description how to the transformation
* Adds a .adoc table to preview _to be consumed by the documentation_
 This table does not longer use paths to coding but xrefs to the service descriptions.

Note that I highly prefer that this folder gets transported/copied to the `docs-stable-5.0` branch before we go final which makes it easier to be consumed by the documentation as we do with other sources coming from ocis. The suggestion for the path is to use the same as we have for e.g. extended envvars but use a new folder like `env-var-deltas`.

In addition, we should take care on naming the files (.md and .adoc) like:
`added-4.0.5-5.0.0.adoc` and `removed-4.0.5-5.0.0.adoc`

### CODE REVIEW

File: docs/env-var-deltas/make-adoc-table-from-md.txt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+add on top under the page heading:
+[width="100%",cols="~,~,~,~",options="header"]
+|===
+
+and at the bottom
+|===
+
+check SSE_HTTP_ADDR and SSE_HTTP_ROOT, there is one | too much
+check WEB_OPTION_LOGIN_URL, there is one | missing
+check WEB_OPTION_DISABLED_EXTENSIONS, in the linen before, there is one | too much
```
Comment (phil-davis) [inline docs/env-var-deltas/make-adoc-table-from-md.txt:18]:
```suggestion
check WEB_OPTION_DISABLED_EXTENSIONS, in the line before, there is one | too much 
```

File: docs/env-var-deltas/make-adoc-table-from-md.txt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+and at the bottom
+|===
+
+check SSE_HTTP_ADDR and SSE_HTTP_ROOT, there is one | too much
+check WEB_OPTION_LOGIN_URL, there is one | missing
+check WEB_OPTION_DISABLED_EXTENSIONS, in the linen before, there is one | too much 
+
+
+sed -i "s/\;/\` \+\n\`/g" 4.0.5-5.0.0.adoc		make each envvar a single line for ease of readbility
```
Comment (phil-davis) [inline docs/env-var-deltas/make-adoc-table-from-md.txt:21]:
```suggestion
sed -i "s/\;/\` \+\n\`/g" 4.0.5-5.0.0.adoc		make each envvar a single line for ease of readability
```

**Review by phil-davis — COMMENTED** [review]
Just a couple of typos to start the review process.

File: docs/env-var-deltas/4.0.5-5.0.0.adoc [hunk 3]
```diff
@@ -0,0 +1,872 @@
+# Added Variables between oCIS 4.0.5 and oCIS 5.0.0-rc
```
Comment (mmattel) [inline docs/env-var-deltas/4.0.5-5.0.0.adoc:1]:
```suggestion
// # Added Variables between oCIS 4.0.5 and oCIS 5.0.0-rc
// commenting the headline to make it better includable
```

**Review by dragonchaser — APPROVED** [review]

**Pull request merged**